src/pages/team.js

```diff
@@ -3,7 +3,7 @@ import { useStaticQuery, graphql } from "gatsby";
 
 import Layout from "../components/layout";
 import SEO from "../components/seo";
-import InfoBlock from "../components/info_block";
+import TeamPage_InfoBlock from "../components/teampage_infoblock";
```

Comment:
Do you think it would be nicer to put the new `InfoBlock` inside a new folder?
e.i. have `import TeamPage from "../components/team/info_block"; ` instead

Reply:
Unless we're going to make more infoblocks I reckon it would be fine just to leave it? But up to you :)